cmd/openshift-install/log.go

```diff
@@ -68,6 +68,9 @@ func setupFileHook(baseDir string) func() {
 	}))
 
 	logrus.Debugf(version.String)
+	if version.Commit != "" {
+		logrus.Debugf("built from commit %s", version.Commit)
```

Comment:
```
$ rg 'logrus.Debug'
cmd/openshift-install/log.go
70:     logrus.Debugf(version.String)

cmd/openshift-install/graph.go
85:             logrus.Debugf("adding node %s", name)
89:             logrus.Debugf("adding edge %s -> %s", name, parent)

cmd/openshift-install/create.go
277:                            logrus.Debugf("Still waiting for the Kubernetes API: %v", err)
281:                            logrus.Debugf("Still waiting for the Kubernetes API: %v", err)
313:                            logrus.Debugf("Failed to list events: %s", err)
319:                            logrus.Debugf("Failed to watch events: %s", err)
334:                            logrus.Debugf("added %s: %s", ev.Name, ev.Message)
337:                            logrus.Debugf("error %s: %s", ev.Name, ev.Message)
375:                                    logrus.Debug("Cluster is initialized")
379:                                    logrus.Debugf("Still waiting for the cluster to initialize: %v",
384:                    logrus.Debug("Still waiting for the cluster to initialize...")
423:                            logrus.Debugf("Route found in openshift-console namespace: %s", route.Name)
428:                    logrus.Debug("OpenShift console route is created")
433:                            logrus.Debugf("Still waiting for the console route: %v", err)
439:                            logrus.Debug("Still waiting for the console route...")

pkg/rhcos/builds.go
50:     logrus.Debugf("Fetching RHCOS metadata from %q", url)
82:     logrus.Debugf("Fetching RHCOS builds from %q", url)

pkg/terraform/terraform.go
46:     tDebug := &lineprinter.Trimmer{WrappedPrint: logrus.Debug}
77:     tDebug := &lineprinter.Trimmer{WrappedPrint: logrus.Debug}
118:    tDebug := &lineprinter.Trimmer{WrappedPrint: logrus.Debug}
154:            logrus.Debugf("Symlinking plugin %s src: %q dst: %q", name, execPath, dst)

pkg/asset/asset.go
65:     logrus.Debugf("Purging asset %q from disk", asset.Name())

pkg/tfvars/libvirt/cache.go
76:             logrus.Debugf("Using cached OS image %q", imagePath)
111:    logrus.Debugf("Unpacking OS image into %q...", imagePath)

pkg/asset/store/store.go
196:    logrus.Debugf("%sFetching %q...", indent, a.Name())
211:            logrus.Debugf("%sReusing previously-fetched %q", indent, a.Name())
225:    logrus.Debugf("%sGenerating %q...", indent, a.Name())
236:    logrus.Debugf("%sLoading %q...", indent, a.Name())
287:                    logrus.Debugf("%sLoading %q from both state file and target directory", indent, a.Name())
293:                            logrus.Debugf("%sOn-disk %q matches asset in state file", indent, a.Name())
312:            logrus.Debugf("%sUsing %q loaded from target directory", indent, a.Name())
317:            logrus.Debugf("%sUsing %q loaded from state file", indent, a.Name())

vendor/github.com/sirupsen/logrus/logger.go
74://      Level: logrus.DebugLevel,

pkg/asset/installconfig/aws/basedomain.go
36:             logrus.Debugf("listing AWS hosted zones (page %d)", i)
```

I think it would be better if Debug statements start with Capital letter.

Reply:
> I think it would be better if Debug statements start with Capital letter.

Done with 5bfca58e1 -> d56fca96e.